crates/transaction-pool/src/config.rs

```diff
@@ -12,6 +14,8 @@ pub struct PoolConfig {
     pub queued_limit: SubPoolLimit,
     /// Max number of executable transaction slots guaranteed per account
     pub max_account_slots: usize,
+    /// The address of the pool owner.
+    pub coinbase: Address,
```

Comment:
hmm I don't think this makes sense here?
what's the argument for binding the coinbase to the pool?

Reply:
I'd expect this to be part of the EthApi.

Reply:
I understand that the _coinbase_ would be the _beneficiary_ or _fee recipient_ of minted blocks, so it would be needed for execution payload building. Maybe we could make it part of `EthApi`, and pass the value to the block builder as a parameter when needed?